scilpy/image/datasets.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from dipy.core.interpolation import trilinear_interpolate4d, \
+                                    nearestneighbor_interpolate
+
+
+class AccessibleVolume(object):
```

Comment:
I agree I think we don't need Accessible

Reply:
I agree. `Volume` is very generic and it might not be clear what it means. However it is in the `dataset` module so it might be clear enough. Other ideas: `DataVolume`? `VoxelGrid`?